.github/workflows/test_backend_uv.yaml

```diff
@@ -32,3 +33,16 @@ jobs:
 
       - name: Run backend integration tests
         run: AUTO_TEST_RUN=y make test-backend-integration
+
+      - name: Collect Ray logs in case of failure
+        if: failure()
+        run: |
+          mkdir /tmp/raylogs
+          docker cp lumigator-ray-1:/tmp/ray/ /tmp/raylogs/
+
+      - name: Upload Ray logs in case of failure
```

Comment:
what are we thinking of doing with these logs? right now they just go to the test runner, i'd avoid in this case since it slows down tests

Reply:
This only runs in case of failure. It is very useful to debug obscure issues like the "no space left on container" one. Currently it's about 100kb zipped, and kept with the test run.